WooCommerce/Classes/POS/Presentation/ItemRowView.swift

```diff
@@ -42,39 +41,45 @@ struct ItemRowView: View {
                 Button(action: {
                     onItemRemoveTapped()
                 }, label: {
-                    Image(systemName: "x.circle")
+                    HStack {
+                        Spacer()
+                        Image(uiImage: UIImage.posRemoveCartItemImage)
+                        Spacer()
+                    }
                 })
                 .frame(width: Constants.buttonWidth,
                        height: Constants.buttonWidth,
                        alignment: .trailing)
-                .padding(.horizontal, Constants.horizontalPadding)
                 .foregroundColor(Color.posIconGrayi3)
             }
         }
         .frame(maxWidth: .infinity, idealHeight: Constants.productCardHeight)
-        .background(Color.posBackgroundGreyi3)
+        .background(Color.posBackgroundWhitei3)
         .overlay {
             RoundedRectangle(cornerRadius: Constants.productCardCornerRadius)
                 .stroke(Color.black, lineWidth: Constants.nilOutline)
         }
         .clipShape(RoundedRectangle(cornerRadius: Constants.productCardCornerRadius))
         .padding(.horizontal, Constants.horizontalPadding)
+        .padding(.vertical, Constants.verticalPadding)
+        .shadow(color: Color.black.opacity(0.08), radius: 4, y: 2)
     }
 }
 
 private extension ItemRowView {
     enum Constants {
-        static let productCardHeight: CGFloat = 64
+        static let productCardHeight: CGFloat = 72
         static let productCardCornerRadius: CGFloat = 8
         // The use of stroke means the shape is rendered as an outline (border) rather than a filled shape,
         // since we still have to give it a value, we use 0 so it renders no border but it's shaped as one.
         static let nilOutline: CGFloat = 0
-        static let horizontalPadding: CGFloat = 32
+        static let verticalPadding: CGFloat = 4
+        static let horizontalPadding: CGFloat = 16
         static let horizontalCardSpacing: CGFloat = 0
         static let horizontalElementSpacing: CGFloat = 16
         static let buttonWidth: CGFloat = 56
-        static let itemNameFont: Font = .system(size: 16, weight: .medium, design: .default)
-        static let itemPriceFont: Font = .system(size: 16, weight: .light, design: .default)
+        static let itemNameFont: Font = .system(size: 16, weight: .semibold, design: .default)
```

Comment:
Observation: there's a space between an item name and the price.

![image](https://github.com/user-attachments/assets/3f2d3d26-fdf4-4972-91f1-ec8227aeeafb)

Reply:
Thanks for the change 👍